packages/api/src/controllers/asset.ts

```diff
@@ -345,6 +346,28 @@ export function getPlaybackUrl(
   return undefined;
 }
 
+export async function getThumbsVTTUrl(
+  asset: WithID<Asset>,
+  os: ObjectStore
+): Promise<string> {
+  const vttUrl = pathJoin(
+    os.publicUrl,
+    asset.playbackId,
+    "thumbnails",
+    "thumbnails.vtt"
+  );
+
+  let params = {
+    method: "HEAD",
+    timeout: 5 * 1000,
+  };
+  let resp = await fetchWithTimeout(vttUrl, params);
```

Comment:
Why do you need to make this HTTP call here? Can't you just return `vttUrl`?

Reply:
Because the vtt will not always be present, it's not switched on for all users yet.

Reply:
Tidied this up now to check the returned files on the asset